Bottom: c673231681df800f72b1207f788a221798187e36
Top:    6cd89b5de76c63a597906aafa913d94cf57ddde4
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-05-07 17:05:25 +0530

Refresh of debug-bits-v2

---

diff --git a/kernel/sched/core.c b/kernel/sched/core.c
index 582c18265c98..2af40d20d7dd 100644
--- a/kernel/sched/core.c
+++ b/kernel/sched/core.c
@@ -2979,7 +2979,6 @@ void wake_up_new_task(struct task_struct *p)
 		per_cpu(nr_lat_sensitive, target_cpu)++;
 		if (sched_feat(MY_DEBUG))
 			trace_printk("wakeup new: pid=%d target_cpu=%d dic=%d\n", p->pid, target_cpu, per_cpu(nr_lat_sensitive, target_cpu));
-		task_unlock(p);
 	}
 #endif
